Samples/runtime/simulator-with-overrides/Operations.qs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Samples.SimulatorWithOverrides {
+    open Microsoft.Quantum.Diagnostics;
+    open Microsoft.Quantum.Intrinsic;
+
+    /// # Summary
+    /// Run a series of experiments that on a perfect state simulator produce matching measurement results.
+    /// If executed on a simulator which introduces errors in measurements, 
+    /// a certain percentage of experiments will produce mismatched results.
+    operation DoCorrelatedMeasurements () : Unit {
+        let nRuns = 100;
+        mutable nSame = 0;
+        for (i in 1 .. nRuns) {
+            using ((q1, q2) = (Qubit(), Qubit())) {
```

Comment:
If you decide to redefine the error model to allow for faulty measurements, then the qubits as allocated at this point in the program may no longer be in the correct state; in that case, you'll prepare one of the other four Bell pairs, rather than the intended (|00⟩ + |11⟩) / √2. As a result, the behavior at line 22 may be different than expected; a comment to that effect may be helpful.

Reply:
I kept the model that keeps measurement results in sync with the state, so this should not be the case. I'm positively looking forward to people experimenting with this kind of behavior on different models :-)

Reply:
Sounds good, hopefully this will spur enough exploration that someone will want to contribute an open systems simulator!